docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+
+volumes:
+  build: ~
+  deps: ~
+
+services:
+  cabbage:
+    image: elixir:1.4
+    entrypoint: mix
+    command: do local.hex --force, deps.get, deps.compile, test
```

Comment:
`test` will run all tests, should we mark this with `--only integration` so it only runs the ones that were tagged with `@tag :integration`? https://github.com/cabbage-ex/cabbage/blob/master/lib/cabbage/feature.ex#L184